openssl-sys/src/lib.rs

```diff
@@ -1206,15 +1206,15 @@ pub const X509_V_ERR_UNSUPPORTED_EXTENSION_FEATURE: c_int = 45;
 pub const X509_V_ERR_UNSUPPORTED_NAME_SYNTAX: c_int = 53;
 pub const X509_V_OK: c_int = 0;
 
-#[cfg(not(ossl101))]
+#[cfg(not(any(ossl101, libressl)))]
```

Comment:
either we don't or that's not sufficient, without the first commit, on a fresh clone of master I get that: https://gist.github.com/Keruspe/12e48a0432b43196c9a6e5de80cd4698